<launch>
    <rosparam command="load" file="$(find hummingbird_pkg)/config/hummingbird_gt_ppo_params_3rotors.yaml" />
    <!-- Launch the training system -->
    <node pkg="hummingbird_pkg" name="hummingbird_training_gt_ppo_3rotors" type="train_hummingbird_gt_ppo_3rotors.py" output="screen"/> -->
</launch>
